pkg/kpackcompat/clientset.go

```diff
@@ -0,0 +1,69 @@
+package kpackcompat
+
+import (
+	"github.com/pivotal/kpack/pkg/apis/build"
+	"github.com/pivotal/kpack/pkg/client/clientset/versioned"
+	v1alpha1client "github.com/pivotal/kpack/pkg/client/clientset/versioned/typed/build/v1alpha1"
+	"github.com/pivotal/kpack/pkg/client/clientset/versioned/typed/build/v1alpha2"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/discovery"
+
+	// load credential helpers
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	KpackGroupVersionV1alpha2 = build.GroupName + "/v1alpha2"
+)
+
+type ClientSetWrapper struct {
+	RealThing versioned.Interface
```

Comment:
```suggestion
	wrapped versioned.Interface
```
Maybe `wrapped` or `kpackClientSet`?

Reply:
dammit woops missed this one, thought I got all the renames

Reply:
`kpackClientSet` +1